app/redux/node/actions.js

```diff
@@ -30,8 +30,10 @@ export const getMiningStatus = (): Action => async (dispatch: Dispatch): Dispatc
   try {
     const status = await httpService.getMiningStatus();
     dispatch({ type: SET_MINING_STATUS, payload: { status } });
+    return status;
```

Comment:
what is the cause for this addition?

Reply:
In create wallet 'nextAction' function, I need to check the status and not allow redirection to nodeSetup, as per last comment, if you agree with it.

Reply:
After all reviews and changes made is this code still has to be changed?